server/images.go

```diff
@@ -160,12 +185,12 @@ func GetModel(name string) (*Model, error) {
 	}
 
 	model := &Model{
+		Digest:    digest,
 		Name:      mp.GetFullTagname(),
 		ShortName: mp.GetShortTagname(),
-		Digest:    digest,
 		Template:  "{{ .Prompt }}",
-		License:   []string{},
 		Size:      manifest.GetTotalSize(),
+		kvs:       make(map[string]llm.KV),
```

Comment:
this is a map of strings to maps of strings, which is a bit hard on the brain for me, any ideas to flatten this out a bit? What does `metadata.Ref` represent here?

Reply:
it's a map of `metadata.Ref` which is the model this layer references to its key value. here's a concrete example

```go
kvs := map[string]llm.KV{
  "sha256:170370233dd5c5415250a2ecd5c71586352850729062ccef1496385647293868": map[string]any{
    "general.architecture": "llama",
    "general.file_type": 2,
    "general.parameter_count": 7241732096,
    "general.quantization_version": 2,
    ...
  }
}
```